spec/support/playwright_global_setup.js

```
@@ -0,0 +1,34 @@
+      env: { ...process.env, RAILS_ENV: 'test' }
+    });
+    
+    // Precompile assets if needed
+    console.log('🎨 Ensuring assets are compiled...');
+    execSync('RAILS_ENV=test bin/rails assets:precompile', { 
+      stdio: 'inherit',
+      env: { ...process.env, RAILS_ENV: 'test' }
+    });
+    
+    // Clear tmp directories
+    console.log('🧹 Cleaning up temporary files...');
```

ESLint

[error] Unexpected console statement.

(no-console)